Week1/homework/showCurrentTime/showCurrentTime.js

```diff
@@ -0,0 +1,16 @@
+function showCurrentTime() {
+    var today = new Date();
+    var h = today.getHours();
+    var m = today.getMinutes();
+    var s = today.getSeconds();
+    m = checkTime(m);
+    s = checkTime(s);
+    document.getElementById('txt').innerHTML =
+    h + ":" + m + ":" + s;
+    var t = setInterval(showCurrentTime, 500);
```

Comment:
setInterval needs to be called only once. This line can live outside of this function. Also you can omit `t` assignment since you don't use it later on